Assets/Scripts/ButtonManager.prefab

```diff
@@ -0,0 +1,45 @@
+%YAML 1.1
```

Comment:
Prefabs should not be in the scripts folder. Also, this comment:
https://github.com/AndreiBaetica/Dungeoneer/pull/133/files?file-filters%5B%5D=.prefab#r667295445

As an aside, I also see another prefab here called DamageIndicatorParent. It should be moved to the prefabs folder.